### PR TITLE
Convert static styles/properties getter to class fields

### PR DESCRIPTION
**Pull request by aomarks**
- Convert static styles getters to static class fields
- Convert static properties getters to static class fields
- Replace invalid curly quotes with straight quotes in code samples
- Update localization docs for JS

### CODE REVIEW

File: packages/lit-dev-content/site/docs/components/properties.md [hunk 1]
```diff
@@ -97,16 +97,14 @@ The argument to the `@property`  decorators is an [options object](#property-opt
 
 ### Declaring properties in a static properties field
 
-To declare properties in a static `properties` field:
+To declare properties in a static `properties` class field:
```
Comment (AndrewJakubowicz) [inline packages/lit-dev-content/site/docs/components/properties.md:100]: nit: Add class to the heading?

Comment (aomarks) [reply packages/lit-dev-content/site/docs/components/properties.md:100]: Done.

File: packages/lit-dev-content/samples/docs/mixins/highlightable/element-two.ts [hunk 2]
```diff
@@ -4,12 +4,10 @@ import {Highlightable} from './highlightable.js'
...
 @customElement('element-two')
 export class ElementTwo extends Highlightable(LitElement) {
-  static get styles() {
-    return [
-      super.styles || [],
-      css`:host { display: block; }`
-    ];
-  }
+  static styles = [
+    super.styles || [],
```
Comment (justinfagnani) [inline packages/lit-dev-content/samples/docs/mixins/highlightable/element-two.ts:8]:
```suggestion
    super.styles ?? [],
```

**Review by justinfagnani — APPROVED** [review]

**Comment by arthurevans** [comment]
Have we decided that this style is OK now? When we released the first RC, the call was still to use the static getters because static class fields were problematic... somewhere. Yeah, I wish I remembered, too. Maybe @justinfagnani does.

**Comment by aomarks** [comment]
> Have we decided that this style is OK now? When we released the first RC, the call was still to use the static getters because static class fields were problematic... somewhere. Yeah, I wish I remembered, too. Maybe @justinfagnani does.

Static class fields were added in Chrome 72 (January 2019), Firefox 75 (April 2020), Safari 14.1 (May 2021).

So I assume the decision was made because Safari didn't have them yet when the lit.dev docs were first written, but now it does.

**Comment by arthurevans** [comment]
Yep--that was it. Thanks, Al!

**Pull request merged**